opm/parser/share/keywords/000_Eclipse100/R/ROCKNUM

```diff
@@ -0,0 +1 @@
+{"name" : "ROCKNUM" , "sections" : ["REGIONS"], "data" : {"value_type" : "INT"}}
```

Comment:
also please add an integer grid property (at least if you don't want downstream to resort to using the deck directly [using the deck here would be a worse kludge than usual because it ignore the modifier keywords, but it is a thing one does if one is desperate enough ;)])